botx/models/files.py

```diff
@@ -229,36 +264,20 @@ def media_type(self) -> str:
         return self._get_mimetype(self.file_name)
 
     @classmethod
-    def has_supported_extension(cls, filename: str) -> bool:
-        """Check that file extension can be handled by BotX API.
-
-        Arguments:
-            filename: file name to check.
-
-        Returns:
-            Matching result.
-        """
-        file_extension = Path(filename).suffix.lower()
-        return file_extension in BOTX_API_ACCEPTED_EXTENSIONS
-
-    @classmethod
-    def get_ext_by_mimetype(cls, mimetype: str) -> str:
+    def get_ext_by_mimetype(cls, mimetype: str) -> Optional[str]:
         """Get extension by mimetype.
 
         Arguments:
             mimetype: mimetype of file.
 
         Returns:
-            file extension.
-
-        Raises:
-            ValueError: when mimetype is unsupported.
+            file extension or none if mimetype not found.
         """
         for ext, m_type in EXTENSIONS_TO_MIMETYPES.items():
             if m_type == mimetype:
                 return ext
 
-        raise ValueError("`{0}` is unsupported mimetype.".format(mimetype))
+        return None
```

Comment:
Maybe `""`?

Reply:
It's not a mimetype.

Reply:
But `""` is not a valid extension, meanwhile `None` says that ext not found